core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListHelper.java

```diff
@@ -90,6 +97,69 @@ private ListHelper() {
             .collect(Collectors.toList());
     }
 
+    /**
+     * Creates a list of entries under given {@code path} based on list of {@link SimpleListItem}
+     * @param resourceResolver Sling {@link ResourceResolver} instance used to create the list
+     * @param path             JCR path of the items list
+     * @param listItems        List of {@link SimpleListItem}
+     * @return Created page containing list of entries or ${@code null} if page cannot be created
+     * @throws WCMException         if a page cannot be created
+     * @throws PersistenceException if a page cannot be deleted or list item cannot be created
+     */
+    public static Page createList(ResourceResolver resourceResolver, String path, List<SimpleListItem> listItems)
+        throws WCMException, PersistenceException {
+        List<Resource> resources = ListResourceUtils.mapToValueMapResources(listItems);
+        return createList(resourceResolver, path, resources);
+    }
+
+    /**
+     * Creates a list of entries under given {@code path} based on key-value map. Each entry is a separate list item.
+     * @param resourceResolver Sling {@link ResourceResolver} instance used to create the list
+     * @param path             JCR path of the items list
+     * @param values           key-value map
+     * @return Created page containing list of entries or ${@code null} if page cannot be created
+     * @throws WCMException         if a page cannot be created
+     * @throws PersistenceException if a page cannot be deleted or list item cannot be created
+     */
+    public static Page createList(ResourceResolver resourceResolver, String path, Map<String, Object> values)
+        throws WCMException, PersistenceException {
+        List<Resource> listItemsProperties = ListResourceUtils.mapToValueMapResources(values);
+        return createList(resourceResolver, path, listItemsProperties);
+    }
+
+    /**
+     * Creates a list of entries under given {@code path} based on list of {@link Resource}
+     * @param resourceResolver Sling {@link ResourceResolver} instance used to create the list
+     * @param path             JCR path of the items list
+     * @param resources        List of {@link Resource}
+     * @return Created page containing list of entries or ${@code null} if page cannot be created
+     * @throws WCMException         if a page cannot be created
+     * @throws PersistenceException if a page cannot be deleted or list item cannot be created
```

Comment:
The first letter in the value/param/return/throws in the java doc should be capitalized.
 `If a page...`